src/PresentationalComponents/RulesTable/RulesTable.js

```diff
@@ -52,7 +52,7 @@ import { useActionsResolver } from './useActionsResolver';
 import { AccountStatContext } from '../../ZeroStateWrapper';
 import impactingFilter from '../Filters/impactingFilter';
 
-const RulesTable = ({ isTabActive }) => {
+const RulesTable = ({ isTabActive, pathway = '' }) => {
```

Comment:
This will cause the `pathway` to always end up in the options, and I'm not sure if we want that.

It think if we make the lines bellow in the options something like `...(CONST_FOR_OPTIONS ? { CONST_FOR_OPTIONS }) : {},` we should not need to set a default for `pathway`, or workloads and selectedTags.

Reply:
Done!

Reply:
@bastilian linter and tests pass now.  Ok to merge it?